L1Trigger/L1TMuonEndCap/src/PatternRecognition.cc

```diff
@@ -416,8 +403,16 @@ void PatternRecognition::process_single_zone(
           int bx1 = bool(bx_shifter & (1<<1));
           int bx0 = bool(bx_shifter & (1<<0));
 
-          if (bx2 == 0 && bx1 == 1) {  // is lifetime up? (note: drift_time is not being used)
+          // is lifetime up?
```

Comment:
Is this hard-coded for the 2-BX setting, or 3-BX setting, or is it free to float with the "BX window" parameter?